.github/workflows/generate.yml

```diff
@@ -37,8 +37,8 @@ jobs:
         # Add the generated tests using the most recent authors name/email
         git config --global user.name "`git log -1 --pretty=format:'%an'`"
         git config --global user.email "`git log -1 --pretty=format:'%ae'`"
-        git add --force -N k8s/*/gen/*.yaml
-        git add -N k8s/
+        git add --force k8s/*/gen/*.yaml
+        git add k8s/
```

Comment:
Do we need both of these adds? The only new files in the `gen` branch should be at the paths above.

Reply:
In practice, no.
In theory, this makes it a little more robust in case some new file gets added there that should also be included.

It does no harm though (I double-checked locally that it doesn't un-add those files or anything).